Automattic/jetpack
# Sync: Update whitelist/blacklist

Supports #16099

#### Changes proposed in this Pull Request:
* Replace usages of whitelist and blacklist.
* Some PHPCS improvements to pass phpcs-changed.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* pb6Nl-duS-p2

#### Does this pull request change what data or activity we track or use?
No change.

#### Testing instructions:
* Verify tests pass.
* Run a full sync, verify it completes as expected.
* Perform various activities on a site, verify it syncs.
* Bonus points for using filters.

#### Proposed changelog entry for your changes:
* Covered under other PRs.
